### DISCUSSION TITLE
[done] extpay.getUser(): User Object with UniqID / E-Mail

### DISCUSSION THREAD

**Discussion by surtic86** [Glench/ExtPay#37] [General]
Hi there,

It would be nice if we could get a UniqID / E-Mail in the User Object from extpay.getUser() to Identify the Users across Systems.

Thanks

**Comment by Glench** [comment]
Would you mind explaining what you're trying to do so I have a little more context?

**Comment by surtic86** [comment]
Sorry did not seen you have already write me back. Well i would like to have the Option to assign them to the Mixpanel Users. So when they reinstall the Extension or use it from another Profile / Browser i see them as one and not as many.

**Comment by Glench** [comment]
Okay! This has been deployed and you shouldn't need to update the ExtPay library to see it. You just do `user.email` and it will return `null` if there is none.

**Reply by surtic86** [reply]
Oh wow thanks! I will test it later.
